tests/Credentials/AssumeRoleWithWebIdentityCredentialProviderTest.php

```diff
@@ -196,6 +196,43 @@ public function testThrowsExceptionWhenRetrievingAssumeRoleCredentialFails()
         }
     }
 
+    /**
+     * @expectedException \Aws\Exception\CredentialsException
+     * @expectedExceptionMessage Error retrieving web identity credentials: Found 1 error while validating the input provided for the AssumeRoleWithWebIdentity operation:
+    [RoleArn] expected string length to be >= 20, but found string length of 11 (0)
```

Comment:
`@expectedExceptionMessage` will match the start of the exception message, so you can probably just include the first line here, as it's enough to verify you're getting the exception message you want.